service/orchestrator/orchestrator.go

```diff
@@ -108,12 +110,6 @@ func WithExternalMetrics(f func() ([]*assessment.Metric, error)) ServiceOption {
 	}
 }
 
```

Comment:
For MEDINA we need a way to fill the catalogs from the outside, so we need something similar to `WithRequirements`.

Reply:
You mean we get the catalog(s) from the central MEDINA respository, and need to store them in the Orchestrator? I was assuming that we simply pre-define the EUCS in Clouditor. Alternatively, we can simply store it via the CreateCatalog API, right?

Reply:
> You mean we get the catalog(s) from the central MEDINA respository, and need to store them in the Orchestrator? I was assuming that we simply pre-define the EUCS in Clouditor. Alternatively, we can simply store it via the CreateCatalog API, right?

I think we need to retrieve it from the catalog component, but using the `CreateCatalog` API sounds like a reasonable option.